src/main/java/chess/controller/ChessController.java

```diff
@@ -0,0 +1,77 @@
+package chess.controller;
+
+import chess.domain.command.Commands;
+import chess.domain.dto.MoveRequestDto;
+import chess.domain.dto.NameDto;
+import chess.domain.exception.DataException;
+import chess.domain.response.GameResponse;
+import chess.service.ChessService;
+import chess.view.ModelView;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.sql.SQLException;
+
+@Controller
+@RequestMapping("/play")
+public class ChessController {
+
+    private static final Gson GSON = new GsonBuilder().setPrettyPrinting().create();
+    private final ChessService chessService;
+
+    public ChessController(ChessService chessService) {
+        this.chessService = chessService;
+    }
+
+    @GetMapping("")
+    public String play(Model model) throws DataException {
+        model.addAllAttributes(ModelView.historyResponse(chessService.loadHistory()));
+        return "lobby";
+    }
```

Comment:
```suggestion
    @GetMapping("")
    public ModelAndView play() throws DataException {
        final ModelAndView modelAndView = new ModelAndView("lobby");
        modelAndView.addAllObjects(ModelView.historyResponse(chessService.loadHistory()));
        return modelAndView;
    }
```

ModelAndView 를 사용할 수 도 있을 거 같아요.

Reply:
왁 이건 처음보는데 한번 알아보고 적용해보겠습니다 !